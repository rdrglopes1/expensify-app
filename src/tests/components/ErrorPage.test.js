import React from 'react';
import { shallow } from 'enzyme';
import NotFoundPage from '../../components/ErrorPage';

test('should render error page correctly', () => {
    const wrapper = shallow(<NotFoundPage />);
    expect(wrapper).toMatchSnapshot();
});