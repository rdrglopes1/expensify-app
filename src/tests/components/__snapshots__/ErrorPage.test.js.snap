// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render error page correctly 1`] = `
<div>
  404! - 
  <Link
    replace={false}
    to="/"
  >
    Go home
  </Link>
</div>
`;
